index.html

```diff
@@ -10,16 +10,31 @@
 <body>
   <header>
     <h1>Tim Eifler</h1>
+    <h3>Junior Webdeveloper</h3>
     <nav>
       <a href="#contact">Contact</a>
       <a href="#biographie">Biographie</a>
     </nav>
   </header>
+  <figure>
+    <img src="https://source.unsplash.com/random/600x300" alt="">
+    <figcaption>Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+      Explicabo voluptatem est nemo reprehenderit laborum libero quisquam 
+      iure facere, excepturi aperiam, ex totam eaque cumque maxime necessitatibus 
+      repellendus vero, fugit voluptatibus.</figcaption>
+  </figure>
   <section id="contact">
     <h2>Contact</h2>
+    <ul>
+      <p>You want to be friends or contact me?</p>
+      <li>Email: <a href="mailto: tim.eif@protonmail.com"> tim.eif@protonmail.com</a></li>
+      <li><a href="https://ca.linkedin.com/in/tim-eifler">Linkedin</a></li>
+      <li><a href="https://github.com/teifler/tim-eifler">GitHub </a></li>
+    </ul>
```

Comment:
I am not sure but I think the p element should not nested inside the ul element :)

Reply:
Absolutly right thanks for checking and noticing :)